.travis.yml

```diff
@@ -17,6 +14,7 @@ matrix:
 
 before_install:
    - export ALLOW_FAILURE=1; if [ "$TRAVIS_PHP_VERSION" != "hhvm" ] && [ "$TRAVIS_PHP_VERSION" != "nightly" ]; then export ALLOW_FAILURE=0; fi
+   - pecl install mcrypt-1.0.2
```

Comment:
You're going to need `yes '' | pecl install mcrypt-1.0.2` for this to work on Travis-CI, to accept the default values in the compilation prompts.